examples/web_client/package.json

```diff
@@ -0,0 +1,44 @@
+{
+    "name": "prismarine-viewer-standalone",
+    "version": "1.0.0",
+    "description": "prismarine-viewer-standalone",
+    "main": "index.js",
+    "scripts": {
+        "prepare": "webpack",
+        "start": "webpack serve",
+        "prod-start": "node server.js",
+        "build-start": "npm run prepare && npm run prod-start"
+    },
+    "dependencies": {
+        "assert": "^2.0.0",
+        "browserify-zlib": "^0.2.0",
+        "buffer": "^6.0.3",
+        "clean-webpack-plugin": "^3.0.0",
+        "constants-browserify": "^1.0.0",
+        "copy-webpack-plugin": "^7.0.0",
+        "crypto-browserify": "^3.12.0",
+        "events": "^3.2.0",
+        "http-browserify": "^1.7.0",
+        "https-browserify": "^1.0.0",
+        "memfs": "^3.2.0",
+        "mineflayer": "^2.39.2",
+        "net-browserify": "^0.2.4",
+        "os-browserify": "^0.3.0",
+        "path-browserify": "^1.0.1",
+        "prismarine-viewer": "^1.13.0",
```

Comment:
Use the relative path, otherwise it reinstall everything.

Reply:
indeed